apps/src/applab/applab.js

```diff
@@ -1139,6 +1139,9 @@ Applab.reset = function(first) {
   var newDivApplab = divApplab.cloneNode(true);
   divApplab.parentNode.replaceChild(newDivApplab, divApplab);
 
+  var newClass = Applab.isRunning() ? 'running' : 'notRunning';
+  $('#divApplab').removeClass('running notRunning').addClass(newClass);
```

Comment:
I wonder if this would be more clear as:
```
$('#divApplab').toggleClass('running', Applab.isRunning());
$('#divApplab').toggleClass('notRunning', !Applab.isRunning());